components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
```suggestion
export const Button = ({ setProduct, produkt, productToBuy }) => {
```

I would like to name the Button more readable, like "procuctButton".